web/info/src/components/nav/profile-dropdown.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+  import { profile } from '$lib/atp';
+
+  import {
+    Menu,
+    MenuButton,
+    MenuItem,
+    MenuItems,
+    Transition
+  } from '@rgossiaux/svelte-headlessui';
+
+  export let hasSession: boolean;
+
+  $: profileAvatarSrc =
+    $profile && $profile.avatar ? $profile.avatar : '/placeholder-avatar.png';
+</script>
+
+<Menu class="relative">
+  <MenuButton>
+    <img
+      src={profileAvatarSrc}
+      alt="Avatar"
```

Comment:
**suggestion**: we can make this decorative and then add an aria-label along the lines of **Open user menu**, so it's clearer to screen readers what this does.

Reply:
I'll add that later this week, for now it's just got `aria-haspopup` and `aria-controls`